open-api/rest-catalog-open-api.yaml

```diff
@@ -3804,6 +4042,21 @@ components:
             }
           }
 
+    UnprocessableEntityResponse:
+      description:
+        Service can not process entity.
```

Comment:
Nit: `can not` -> `cannot` for consistency in the spec

Reply:
will fix